src/Exception/Contexts/UnsupportedContextKeyException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Nipwaayoni\Exception\Contexts;
+
+class UnsupportedContextKeyException extends \Exception
+{
+    public function __construct(string $message = '', int $code = 0, \Throwable $previous = null)
+    {
+        parent::__construct(sprintf('Unknown context key provided: %s', $message), $code, $previous);
```

Comment:
Instead of joining the keys with `|`, I think that `", "` could be better.

`sprintf('Unknown context key provided: "%s"', join('", "', $keys))`

Assuming that now we have a list of keys as constructor argument.

Reply:
Agreed.

Reply:
Updated as described.